src/main/webapp/WEB-INF/jsp/feedback_form.jsp

```diff
@@ -186,34 +42,24 @@ form.userAgent.value=navigator.userAgent;
             <input type="hidden" name="<c:out value="${status.expression}"/>" />
             </spring:bind>
 
-            <p style='font-size: smaller;'>
-               (limit is 1200 characters)
-            </p>
-
-            <p class='reply'>
-                <strong>Do you want a reply?</strong><br/>
-                <spring:bind path="command.reply">
-                    <input type="radio" name="<c:out value="${status.expression}" />" value="Yes"> Yes, I would like someone to follow up with me.</input><br/>
-                    <input type="radio" name="<c:out value="${status.expression}"/>" value="No"> No thanks.</input>
-                </spring:bind>
+            <p>
+                Your name will be submitted with your feedback.
             </p>
             <p>
-            <input type="submit" value="Send message" class='btn btn-primary btn-share-ideas btn-gen-feedback' />
+            <input type="submit" value="Send" class='btn btn-primary btn-share-ideas' />
             </p>
         </form>
       </div>
       <div class="col-xs-12 col-sm-6 general-help">
-        <h2>Need Help?</h2>
+        <h2>Get help</h2>
           <a href='/portal/p/campus-chat-connections'><i class="fa fa-comments-o"></i>Live chat</a>
           <a href='http://helpdesk.doit.wisc.edu/page.php?id=1' target='_blank'><i class="fa fa-phone"></i>Call Us</a>
           <a href='https://kb.wisc.edu/myuw/' target='_blank'><i class="fa fa-question"></i>How-to Info</a>
       </div>
     </div>
+    <div class="my-voice-container">
+        <span> <img src="<%=renderRequest.getContextPath()%>/img/myuw-myvoice-logo.png"/></span>
+        <span> MyUW MyVoice closed on March 30.  We received hundreds of ideas and are using them to enhance the site.  Thanks to everyone who contributed, and we hope you'll keep giving your feedback on MyUW!</span>
```

Comment:
Year on this? We have a tendency to let things sit for a while. Or add a Jira to remove this in the future?